src/webextension/settings/components/app.css

```diff
@@ -2,6 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+html {
+  /* make sure this fully displays on the about:addons view */
+ flex-direction: column;
```

Comment:
I don't think this should be necessary; there's only one child element anyway.

Reply:
>there's only one child element anyway.

Good point. I think I was playing around with some different attempts and kept this unnecessarily.